Projects/hCoreUI/Sources/hButton/hButton.swift

```diff
@@ -13,10 +13,8 @@ struct LargeButtonModifier: ViewModifier {
 struct MediumButtonModifier: ViewModifier {
     func body(content: Content) -> some View {
         content
-            .padding(.horizontal, 16)
```

Comment:
We should not remove this. We need this to have same paddings everywhere for medium button